### PR TITLE
[caclmgrd] Only add a default DROP rule on the INPUT chain

### PR DESCRIPTION
**Pull request by jleveque**
No longer add a default DROP rule on the FORWARD chain

### CODE REVIEW

File: files/image_config/caclmgrd/caclmgrd [hunk 1]
```diff
@@ -378,9 +378,7 @@ class ControlPlaneAclManager(object):
         # add iptables/ip6tables commands to drop all other incoming packets
         if num_ctrl_plane_acl_rules > 0:
             iptables_cmds.append("iptables -A INPUT -j DROP")
-            iptables_cmds.append("iptables -A FORWARD -j DROP")
```
Comment (lguohan) [inline files/image_config/caclmgrd/caclmgrd:381]: please also remove line 201 and 212

File: files/image_config/caclmgrd/caclmgrd [hunk 2]
```diff
@@ -378,9 +378,7 @@ class ControlPlaneAclManager(object):
         # add iptables/ip6tables commands to drop all other incoming packets
         if num_ctrl_plane_acl_rules > 0:
             iptables_cmds.append("iptables -A INPUT -j DROP")
```
Comment (prsunny) [inline files/image_config/caclmgrd/caclmgrd:380]: Just thinking, does the INPUT rule drop the "forwarding" packets?

File: files/image_config/caclmgrd/caclmgrd (hunk 1 above)
Comment (yxieca) [reply files/image_config/caclmgrd/caclmgrd:381]: Should we add forward rule accept here instead?

Comment (jleveque) [reply files/image_config/caclmgrd/caclmgrd:381]: @yxieca: That shouldn't be necessary as long as the default policy for the chain is ACCEPT, which is what is explicitly being done on lines 201 and 212.

File: files/image_config/caclmgrd/caclmgrd (hunk 2 above)
Comment (prsunny) [reply files/image_config/caclmgrd/caclmgrd:380]: tested this, and INPUT rule drop does not affect the "forwarding" packet. So we should be good

**Pull request merged**